src/Components/ServiceScene/LogsVolumePanel.tsx

```diff
@@ -23,21 +41,59 @@ export class LogsVolumePanel extends SceneObjectBase<LogsVolumePanelState> {
         panel: this.getVizPanel(),
       });
     }
+
+    const fields = sceneGraph.lookupVariable(VAR_FIELDS, this) as AdHocFiltersVariable;
+    const patterns = sceneGraph.lookupVariable(VAR_PATTERNS, this) as CustomVariable;
+    const lineFilter = sceneGraph.lookupVariable(VAR_LINE_FILTER, this) as CustomVariable;
+    const labels = sceneGraph.lookupVariable(VAR_LABELS, this) as AdHocFiltersVariable;
+
+    this._subs.add(
+      fields.subscribeToState((newState, prevState) => {
+        if (newState.filters.length !== prevState.filters.length) {
+          this.updateVolumePanel();
+        }
+      })
+    );
+    this._subs.add(
+      labels.subscribeToState((newState, prevState) => {
+        if (newState.filters.length !== prevState.filters.length) {
+          this.updateVolumePanel();
+        }
+      })
+    );
+    this._subs.add(
+      patterns.subscribeToState((newState, prevState) => {
+        if (newState.value !== prevState.value) {
+          this.updateVolumePanel();
+        }
+      })
+    );
+    this._subs.add(
+      lineFilter.subscribeToState((newState, prevState) => {
+        if (newState.value !== prevState.value) {
+          this.updateVolumePanel();
+        }
+      })
+    );
   }
 
+  public updateVolumePanel = () => {
+    this.setState({
+      panel: this.getVizPanel(),
+    });
+  };
+
   private getVizPanel() {
+    const service = this.service();
+    const queryString = this.hasSingleServiceSelector()
+      ? `sum by (${LEVEL_VARIABLE_VALUE}) (sum_over_time({__aggregated_metric__=\`${service}\`} | logfmt | unwrap count [$__auto]))`
```

Comment:
the `_aggregated_metric__` query is still getting fields injects as line filter statements. I'm surprised that would happen since I'm not using the `$LOG_STREAM_SELECTOR_EXPR`. Where's that coming from, and is there a way to disable it in this case?

Reply:
We use `service_name` as an "AddHocVariable": https://github.com/grafana/explore-logs/blob/main/src/Components/IndexScene/IndexScene.tsx#L180-L189

The way scenes works, it will try to add all those variables as add hoc filters to every query run. We would need to change the variable type in order for this to work. As a workaround for your PR you could change
https://github.com/grafana/explore-logs/blob/main/src/Components/IndexScene/IndexScene.tsx#L182 to something like `datasource: { type: 'fake' },`.
With that you wouldn't see those `service_name` labels included, however it would break showing the values in the label dropdowns.